.github/workflows/build.yml

```
@@ -0,0 +1,24 @@
+name: Build
+on:
+  pull_request:
+    paths:
```

Could also add this workflow definition to the list:
```suggestion
    paths:
      - .github/workflows/build.yml
```